clang/lib/CodeGen/CGGPUBuiltin.cpp

```diff
@@ -70,7 +70,8 @@ RValue
 CodeGenFunction::EmitNVPTXDevicePrintfCallExpr(const CallExpr *E,
                                                ReturnValueSlot ReturnValue) {
   assert(getTarget().getTriple().isNVPTX());
-  assert(E->getBuiltinCallee() == Builtin::BIprintf);
+  assert(E->getBuiltinCallee() == Builtin::BIbuiltin__printf ||
```

Comment:
The function seems to process `printf` built-in already. Can't we use it instead of adding support for `__builtin_printf`?

Reply:
That's a good point, done now.